### PR TITLE
Reset password

### PR DESCRIPTION
**Pull request by hira-iqbal**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -19,7 +19,9 @@
     <div class="container p-3 my-2 bg_dark text-black">
       <nav class="navbar navbar-expand-lg navbar-light bg-light">
         <%= link_to 'Articles', root_path, class: "navbar-brand" %>
-        <%= link_to 'Admin', users_path, class: "navbar-brand", admin: true %>
+        <% if current_user.role =='admin'%>
```
Comment (aliammaar) [inline app/views/layouts/application.html.erb]: I think I told you 3 times to use the instance method from `app/models/user.rb`

**Review by aliammaar — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (hira-iqbal) [reply app/views/layouts/application.html.erb:22]: I updated this file but It was staged mistakenly.

Comment (hira-iqbal) [reply app/views/layouts/application.html.erb]: It was staged mistakenly.

Comment (hira-iqbal) [reply app/views/layouts/application.html.erb]: I have updated this file.

**Review by aliammaar — APPROVED** [review]

**Pull request merged**